Docs/diataxis 2 (#898)
iotaledger/wasp: documentation/docs/guide/core_concepts/validators.md

---
description: Each chain is run by a network of validator nodes which run a consensus on the chain state update.
image: /img/logo/WASP_logo_dark.png
keywords:
- validators
- validator nodes
- consensus

> - state update

Reviewer: ```suggestion
- state update
- explanation
```

Author: ahh, so I did everything except the thing that I was supposed to do.

---
# Validators

> Each chain is run by that chain's *committee of validators*. ISC does not define how do you select validators to form a committee: it could be a solitary choice of the chain's owner, or it could be a [public competition](https://wiki.assembly.sc/learn/introduction/) between candidates. ISC does not define how validators are rewarded, either.

Reviewer: ```suggestion
Each chain is run by that chain's *committee of validators*. ISC does not define how you select validators to form a committee: it could be a choice of the chain's owner or a [public competition](https://wiki.assembly.sc/learn/governance/dynamic_committees) between candidates. ISC does not define how validators are rewarded, either.
```

I'm not sure we have this feature yet.

> Each committee is made of a known finite number of validators, and each validator holds its own part of a common key. On their own, validators cannot meddle with the chain. Together, they have full control over it.

Reviewer: ```suggestion
Each committee is composed of a known finite number of validators, and each validator holds its part of a common key. On their own, validators cannot meddle with the chain. Together, they have complete control over it.
```
Isn't it a shared key?

> Validators own computers that perform the actual calculations to maintain the chain. We call these computers validator nodes, and our software for running a validator node we called Wasp.

Reviewer: ```suggestion
Validators own computers that perform the actual calculations to maintain the chain. These computers are validator nodes, and the software running on the validator node is Wasp.
```

This paragraph seems to be a bit out of context here. Perhaps it'd work better at the beginning.
